waisntechtools/alerts/urls.py

```diff
@@ -6,5 +6,7 @@
 app_name = 'alerts'
 urlpatterns = [
     path('', lambda request: render(request, 'alerts/index.html'), name='index'),
-    path('debug', views.DebugView.as_view(), name='debug')
+    path('debug', views.DebugView.as_view(), name='debug'),
+    path('', views.IndexView.as_view(), name='index'),
+    path('messages/', views.twilio_view, name='messages')
```

Comment:
Since this is essentially a debugging tool, could we add it to the `DebugView`? It would be great if we can move all of the debug Twilio code into the `DebugView` class as well. There is a paradigm in software engineering that code should have **high cohesion and loose coupling**. I think that code right now could have better cohesion.

Reply:
This is something I'm up for!